### PR TITLE
Delete Dockerfile

### PR DESCRIPTION
**Pull request by dairbuirabass**
## Description
Remove Dockerfile

## Related Issue
-

## Motivation and Context
Dockerfile not needed

## Checklist:
- [x] My code follows the code style of this project.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] I have assigned the correct milestone or created one if non-existent.
- [x] I have correctly labeled this pull request.
- [ ] I have linked the corresponding issue in this description.
- [x] I have updated the corresponding Jira ticket.
- [x] I have requested a review from at least 2 reviewers
- [x] I have checked the base branch of this pull request
- [x] I have checked my code for any possible security vulnerabilities

### CODE REVIEW

File: .github/workflows/action.yml [hunk 1]
```diff
@@ -73,19 +73,6 @@ jobs:
         extensions: ast, bcmath, gd
         coverage: none
 
-      #https://github.com/actions/cache/blob/master/examples.md#php---composer
```
Comment (olsi-qose) [inline .github/workflows/action.yml:76]: Why is this deleted? Be careful when deleting steps from the actions

Comment (dairbuirabass) [reply .github/workflows/action.yml]: I made those changes to test if it is causing the inspection runs, removed now. Ready for review @olsi-qose

**Review by olsi-qose — APPROVED** [review]
